app/models/docker_builder_service.rb

```diff
@@ -71,9 +80,12 @@ def build_image(tmp_dir)
     output.puts("### Running Docker build")
 
     build.docker_image =
-      Docker::Image.build_from_dir(tmp_dir, {}, Docker.connection, registry_credentials) do |output_chunk|
-        handle_output_chunk(output_chunk)
-      end
+      Docker::Image.build_from_dir(
+        tmp_dir,
```

Comment:
A thought just occurred to me. We might need to specify what the root directory is for doing the Docker build.

For example, if there's a repo with a file called `$root/scala/Dockerfile`, there's a decent chance the base directory should be `$root/scala`, instead of `$root`.